smoke-test/tests/read_only/test_search.py

```diff
@@ -1,25 +1,55 @@
 import pytest
-import requests
 
 from tests.test_result_msg import add_datahub_stats
-from tests.utils import get_frontend_url, get_gms_url
+from tests.utils import get_frontend_session, get_frontend_url
 
 restli_default_headers = {
     "X-RestLi-Protocol-Version": "2.0.0",
 }
 
+ENTITY_TO_MAP = {
```

Comment:
is there a programmatic way to populate this mapping?

Reply:
Not as far as I am aware